src/components/Badge.tsx

```diff
@@ -17,26 +17,22 @@ along with the library. If not, see <http://www.gnu.org/licenses/>.
 */
 
 import { colord } from 'colord'
-import { ReactNode } from 'react'
 import { StyleProp, View, ViewStyle } from 'react-native'
 import styled, { css } from 'styled-components/native'
 
 interface BadgeProps {
-  children: ReactNode
   color?: string
   rounded?: boolean
   border?: boolean
   light?: boolean
   style?: StyleProp<ViewStyle>
 }
 
-const Badge = ({ style, color, children }: BadgeProps) => {
-  return (
-    <View style={style}>
-      <BadgeText color={color}>{children}</BadgeText>
-    </View>
-  )
-}
+const Badge: FC<BadgeProps> = ({ style, color, children }) => (
```

Comment:
Why are we back to using FC :(

Reply:
I thought it was a mistake that we forgot to update so I changed everything :D